core/statistics/softwareVersion/softwareVersion.go

```diff
@@ -0,0 +1,72 @@
+package softwareVersion
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/core/appStatusPolling"
+)
+
+const checkInterval = time.Hour
+const stableTagLocation = "https://api.github.com/repos/ElrondNetwork/elrond-go/releases/latest"
+
+type tagVersion struct {
+	TagVersion string `json:"tag_name"`
+}
+
+type softwareVersionChecker struct {
+	statusHandler core.AppStatusHandler
+}
+
+// NewSoftwareVersionChecker will create a object for software  version checker
+func NewSoftwareVersionChecker(appStatusHandler core.AppStatusHandler) (*softwareVersionChecker, error) {
+	if appStatusHandler == nil || appStatusHandler.IsInterfaceNil() {
+		return nil, appStatusPolling.ErrNilAppStatusHandler
```

Comment:
declare the error in the same package. No need to import another package just for this